tests/test_data/cropA/pyrate_mexico_cropa.conf

```diff
@@ -122,6 +125,24 @@ orbfitdegrees: 1
 orbfitlksx:    1
 orbfitlksy:    1
 
+# phase closure params
+# large_deviation_threshold_for_pixel # pi/2, pi=3.1416
+# threshold_to_remove_ifg: ifgs with more than this fraction of pixels with error will be dropped
+# loop_count_for_threshold_to_remove_ifg: pixel with phase unwrap error in at least this many loops
+# phase_unwrap_error_threshold: pixel with phase unwrap error in more than this many ifgs will be flagged
+# max_loop_length: loops upto this many edges are considered for closure checks
+# subtract_median_in_closure_check: whether to subtract median during closure checks
+# max_loop_count_for_each_ifgs: loops with more than these many ifgs are discarded.
+# max_loop_count_for_each_ifgs: Ifg count must be met for all ifgs in the loop for loop to be discarded
+large_dev_thr: 1.5708
+threshold_to_remove_ifg: 0.07
+loop_count_for_threshold_to_remove_ifg: 2
+phase_unwrap_error_threshold: 5
+max_loop_length: 4
+subtract_median_in_closure_check: 1
+max_loop_count_for_each_ifgs: 2
+
+
```

Comment:
Please add to the top level `input_parameters.conf` - this config file is more visible to users

Reply:
resolved by https://github.com/GeoscienceAustralia/PyRate/pull/307/commits/9f840bc6d9ffd818c5e7cf6c99a91be12696403a